src/Net/LinkShortener.php

```diff
@@ -29,7 +30,7 @@ public static function Get(string $shortLink): ?string {
 
     public static function Create(string $longUrl): string {
         $redis = self::GetRedisInstance();
-        while($key = self::GenerateShortString()) {
+        while($key = self::GenerateShortString($longUrl)) {
```

Comment:
This while has to go away now since $key is expected to be a duplicate when the same $longUrl is given.
It should just get the key and set it.